content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -0,0 +1,119 @@
+    $ kind get clusters
+    istio-testing
+    {{< /text >}}
+
+1.  You can also check list of available local cluster contexts using following command.
+
+    {{< text bash >}}
+    $ kubectl config get-contexts
+    CURRENT   NAME                 CLUSTER              AUTHINFO             NAMESPACE
+    *         kind-istio-testing   kind-istio-testing   kind-istio-testing
+              minikube             minikube             minikube
+    {{< /text >}}
+
+    {{< tip >}}
+    `kind` is automatically prefixed in cluster name `kind-istio-testing`
+    {{< /tip >}}
+
+1.  If you run multiple clusters, you need to choose which cluster `kubectl` talks to. You can set a default cluster
+    for `kubectl` by setting the current context in [Kubernetes kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file. Additionally you can run following command
+    to set the current context for `kubectl`.
+
+    {{< text bash >}}
+    $ kubectl config use-context kind-istio-testing
+    Switched to context "kind-istio-testing".
+    {{< /text >}}
+
+    Once you are done setting up local cluster, you can proceed to [setup Istio](/docs/setup/getting-started/#download)
+    on it.
+
+1.  When you are done experimenting and you want to delete existing cluster, use the following command:
+
+    {{< text bash >}}
+    $ kind delete cluster --name istio-testing
+    Deleting cluster "istio-testing" ...
+    {{< /text >}}
+
+## Setup Dashboard UI for kind
+
+kind does not have in built dashboard UI like minikube. But you can still setup and view your cluster.
+Follow these instruction to setup dashboard UI for kind.
+
+1.  The dashboard UI is not deployed by default. To deploy it, run the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-beta8/aio/deploy/recommended.yaml
+    {{< /text >}}
+
+1.  Verify that the dashboard is deployed and running.
+
+    {{< text bash >}}
+    $ kubectl get pod -n kubernetes-dashboard
+    NAME                                         READY   STATUS    RESTARTS   AGE
+    dashboard-metrics-scraper-76585494d8-zdb66   1/1     Running   0          39s
+    kubernetes-dashboard-b7ffbc8cb-zl8zg         1/1     Running   0          39s
+    {{< /text >}}
+
+1.  Create `ClusterRoleBinding` to provide admin access to the newly created cluster.
+
+    {{< text bash >}}
+    $ kubectl create clusterrolebinding default-admin --clusterrole cluster-admin --serviceaccount=default:default
+    {{< /text >}}
+
+1.  In order to login to the dashboard UI you need a token. Use following command to store token in variable.
+
+    {{< text bash >}}
+    $ token=$(kubectl get secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='default')].data.token}"|base64 -d)
+    {{< /text >}}
+
+    Display token using the echo command and copy it so you can use it to login to dashboard.
```

Comment:
Display **the** token using the `echo` command and copy it to use for logging into Dashboard.